### PR TITLE
Add back deprecated bitclout fields for backwards compatability

### PR DESCRIPTION
**Pull request by maebeam**

### CODE REVIEW

**Comment by AeonSw4n** [comment]
LGTM )

**Comment by chafreaky** [comment]
Will this include all previous responses or only the ones addressed so far?

**Comment by maebeam** [comment]
@chafreaky what responses am I missing? I tried to get all of them

**Comment by chafreaky** [comment]
@maebeam specifically the CoinEntry ones (I see you took care of some) - I don't see one for "DeSoLockedNanos". Might be missing it though.

**Comment by maebeam** [comment]
Thanks! Just added support for `BitCloutLockedNanos`

File: routes/post.go [hunk 1]
```diff
@@ -207,8 +208,8 @@ func (fes *APIServer) _postEntryToResponse(postEntry *lib.PostEntry, addGlobalFe
 		LikeCount:                      postEntry.LikeCount,
 		DiamondCount:                   postEntry.DiamondCount,
 		CommentCount:                   postEntry.CommentCount,
-		RepostCount:                   postEntry.RepostCount,
-		QuoteRepostCount:              postEntry.QuoteRepostCount,
+		RepostCount:                    postEntry.RepostCount,
```
Comment (lazynina) [inline routes/post.go:211]: We should add back RecloutCount and QuoteRecloutCount to maintain backwards compatibility

**Review by lazynina — CHANGES_REQUESTED** [review]
Can we also add back routes that have replaced bitclout or clout with deso in the path? There are just a few I think - mainly in admin_buy_deso

File: routes/post.go (hunk 1 above)
Comment (maebeam) [reply routes/post.go:211]: Ty!

**Pull request merged**